unity/Assets/Scripts/AgentManager.cs

```diff
@@ -1365,6 +1366,13 @@ public SimObjType ReceptableSimObjType()
 		return (SimObjType)Enum.Parse(typeof(SimObjType), receptacleObjectType);
 	}
 
+    public VisibilityScheme GetVisibilityScheme() {
+        if (string.IsNullOrEmpty(visibilityScheme)) {
+            return VisibilityScheme.Collider;
+        }
+		return (VisibilityScheme)Enum.Parse(typeof(VisibilityScheme), visibilityScheme);
+    }
+
```

Comment:
Rather than have null/empty represent `Collider`, why not have `visibilityScheme` have a default value of `"Collider"`?

Reply:
I updated this to default to "Collider", but I also added some error handling if Enum can't parse the passed in param.  The original motivation was to handle the case if someone sent in an empty string for visibilityScheme, but checking for parse errors is more robust.